presto-docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -259,6 +261,266 @@ Field           Required  Type      Description
 
 There is no limit on field descriptions for either key or message.
 
+Kafka inserts
+-------------
+
+The Kafka connector supports the use of SQL ``INSERT INTO`` statements to write data to a Kafka topic.
+Table column data is mapped to Kafka messages as defined in the table definition file, `Table Definition Files`_.
```

Comment:
https://developers.google.com/style/link-text .. so in this case set an anchor and the use :ref:`in the table definition file <my-anchor>` .. ping me on slack if you want to edit together